### ISSUE TITLE
Id conflicts with the Id of an existing relationship.

### ISSUE THREAD

**Issue by a-gerold** [ClosedXML/ClosedXML#221]
Hi all,

im using ClosedXML 0.86.0.0 as NuGet in a C# Controller.

I have the problem, that if i am adding comments to cells, i can not save the workbook to a MemoryStream.
It throws a system ArgumentException: "Id conflicts with the Id of an existing relationship.
Parametername: id"

If I remove the line with cell.Comment..... no Exception is thrown.

                        `// this causes the problem
                        cell.Comment.AddText(string.Join(", ", hourBookings.Select(b => b.Description)));
                        currRow++;
                    }

                    currWorksheet.ColumnsUsed().AdjustToContents();
                }
            }

            workbook.SaveAs(@"C:\temp\test2.xlsx", false); <- Works as it should
            
            using (var stream = new MemoryStream())
            {
                workbook.SaveAs(stream, false); <---- throws System.ArgumentException`

**Comment by igitur** [comment]
Please post full code to reproduce the issue.

**Comment by a-gerold** [comment]
I found out, that the problem also disappears if I do not save the workbook as file before i save to the memory stream.
However, you should be able to reproduce with the following code:

```C#
public void TestWriteToMemory()
{
    var wb = new XLWorkbook();
    var sheet = wb.AddWorksheet("one");
    var c = sheet.Cell(1, 1);

    c.Comment.AddText("abc"); // Comment this or the next line out, for no Exception
    wb.SaveAs(@"C:\temp\testxl.xlsx", false); // Comment and also SaveAs File cause Argument Exception

    using (var stream = new MemoryStream())
    {
        wb.SaveAs(stream, false);
    }
}
```

**Comment by igitur** [comment]
Thanks for reporting. This will be fixed in v0.87

**Comment by igitur** [comment]
The PR has been merged into the `develop` branch. I'd appreciate it if you could pull the `develop` branch and check that it solves your issue. Thanks.
